Fhi.HelseId.Samples/HostBuilders/TestHostBuilderSamples.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net;
+using Fhi.HelseId.Web.ExtensionMethods;
+using Fhi.TestFramework.Extensions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NUnit.Framework;
+
+namespace Fhi.Samples.TestServerSetup
+{
+    /// <summary>
+    /// Using HostBuilder with TestServer
+    /// </summary>
+    public class TestServerSamples
+    {
+        [Test]
+        public async Task UsingHostBuilder()
```

Comment:
Can this test name be improved?

Reply:
This is not really a test just a sample so I was a bit uncertain what to call it. Any suggestion?

Reply:
Next step when someone suggest a better name for samples.